### PR TITLE
Allow for boolean arguments

### PR DESCRIPTION
**Pull request by auchEnia**
added callback function to output the errors in terminal/console by default;
arguments can be passed along with the cordova platform specific commands (e.g. --release);

### CODE REVIEW

File: wrapper.js [hunk 1]
```diff
@@ -20,6 +20,14 @@ function cordovaWrapper(commandArray, next) {
   var msg
   var known_platforms = Object.keys(cordova_lib.cordova_platforms)
 
+  /**
+   * Output the error in terminal/console
+   * @param dbg_msg
+   */
+  var callback = function (dbg_msg) {
```
Comment (rcsole) [inline wrapper.js]: Would you mind using the form?
```js
function callback() {
  // ...
}
```

**Comment by rcsole** [comment]
Looks good, thanks for the contribution! Would you mind changing what I mentioned?

**Comment by auchEnia** [comment]
No problem, happy to help. Your plugin helped us a lot. I just pushed new commit to follow the coding style.

**Comment by rcsole** [comment]
Published to NPM and added you to collaborators! Thanks!

**Pull request merged**